developer-workflow/c-api.rst

```diff
@@ -4,18 +4,20 @@
 Changing Python's C API
 =======================
 
-The C API is divided into three sections:
+The C API is divided into these tiers:
 
 1. The internal, private API, available with ``Py_BUILD_CORE`` defined.
    Ideally declared in ``Include/internal/``. Any API named with a leading
    underscore is also considered private.
-2. The public C API, available when ``Python.h`` is included normally.
+2. The “general” public C API, available when ``Python.h`` is included normally.
    Ideally declared in ``Include/cpython/``.
-3. The Limited API, available with ``Py_LIMITED_API`` defined.
+3. The Unstable C API, identified by the ``PyUnstable_`` name prefix.
+   Ideally declared in ``Include/cpython/`` along with the general public API.
+4. The Limited C API, available with ``Py_LIMITED_API`` defined.
```

Comment:
```suggestion
4. The Limited C API, available with :c:macro:`Py_LIMITED_API` defined.
```

Thanks to the intersphinx support we added, you can auto-format and cross-reference this macro directly here.